Code/src/main/java/nl/utwente/viskell/ghcj/GhciEvaluator.java

```diff
@@ -62,6 +62,12 @@ print a zero byte whenever it expects the user (that's us) to enter
            useful sentinel. */
         this.eval(":set prompt " + SENTINEL);
 
+        /* Load some useful libraries. */
+        this.eval("import Prelude");
+        this.eval("import Data.List");
+        this.eval("import Data.Maybe");
+        this.eval("import Data.Either");
```

Comment:
This can also be replaced by a single _":m Prelude Data.List Data.Maybe Data.Either"_.
Or even better, no literals, but just concatenating a(n observable) list of required modules.